plenoptic/metric/perceptual_distance.py

```diff
@@ -66,7 +66,7 @@ def _ssim_parts(img1, img2, dynamic_range):
 
     real_size = min(11, height, width)
     std = torch.tensor(1.5).to(img1.device)
-    window = circular_gaussian2d(real_size, std=std, out_channels=n_channels)
+    window = circular_gaussian2d(real_size, std, n_channels, img1.device)
```

Comment:
is adding a device arg better than calling `.to(device)` on the resulting tensor?

Reply:
especially because it doesn't look like we're consistent: `models.naive.Gaussian` doesn't accept a device arg. Is the idea that models don't accept a device arg, but components can?

Reply:
removed device arg in favour of better device inference within the `circular_gaussian2d` function